tools/generate_tables_sql_and_testNames.php

```diff
@@ -23,95 +26,118 @@
 
 $data = stream_get_contents(STDIN);
 
-$instruments=explode("{-@-}",trim($data));
-
-$tblCount=0;
-$parameterCount=0;
-$pages = array();
-foreach($instruments AS $instrument){
-    $catId="";
-    $items=explode("\n",trim($instrument));
-    foreach($items AS $item){
-        $paramId="";
-        $bits=explode("{@}",trim($item));
-        if(preg_match("/Examiner[0-9]*/" , $bits[1])){
+$instruments =explode("{-@-}", trim($data));
+
+$tblCount       = 0;
+$parameterCount = 0;
+$pages          = [];
+foreach ($instruments as $instrument) {
+    $catId = "";
+    $items =explode("\n", trim($instrument));
+    foreach ($items as $item) {
+        $paramId = "";
+        $bits    =explode("{@}", trim($item));
+        if (preg_match("/Examiner[0-9]*/", $bits[1])) {
             continue;
         }
-        switch($bits[0]){
-            //generate the CREATE TABLE syntax
-            case "table":
-                $tablename = $bits[1];
-
-                $filename="../project/tables_sql/".$bits[1].".sql";
-                $output="CREATE TABLE `$bits[1]` (\n";
-                $output.="`CommentID` varchar(255) NOT NULL default '',
-                          `UserID` varchar(255) default NULL,
-                          `Examiner` varchar(255) default NULL,
-                          `Testdate` timestamp DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
-                          `Data_entry_completion_status` enum('Incomplete','Complete') NOT NULL default 'Incomplete',\n";
+        switch ($bits[0]) {
+        // generate the CREATE TABLE syntax
+        case "table":
+            $tablename = $bits[1];
+            $filename  = "../project/tables_sql/".$bits[1].".sql";
+            $stat      = [ 'Incomplete', 'Complete' ];
+            $output   .= "CREATE TABLE `$bits[1]` (
+    `CommentID` varchar(255) NOT NULL default '',
+    `UserID` varchar(255) default NULL,
+    `Examiner` varchar(255) default NULL,
+    `Testdate` timestamp DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+    `Data_entry_completion_status` enum(
+	$stat.join(',')
```

Comment:
This looks wrong. 1. That's halfway between javascript and php syntax and 2. It looks likeit's code embedded in the middle of a string..